dagql/idtui/frontend_pretty.go

```diff
@@ -158,10 +158,6 @@ func (fe *frontendPretty) Shell(ctx context.Context, handler ShellHandler) {
 	<-ctx.Done()
 }
 
-func (fe *frontendPretty) ConnectedToEngine(ctx context.Context, name string, version string, clientID string) {
-	// noisy, so suppress this for now
-}
```

Comment:
This seems to be a relic of before we really had `Enscapsulated` spans. We can how have our cake (have an OTEL span) and eat it too (not show it to the user by default) :grin: